tests/test_enterprise/api/test_views.py

```diff
@@ -77,18 +77,15 @@
     kwargs={'pk': FAKE_UUIDS[1]}
 )
 ENTERPRISE_CATALOGS_COURSE_ENDPOINT = reverse(
-    # pylint: disable=anomalous-backslash-in-string
-    r'enterprise-catalogs-courses/(?P<course-key>[^/+]+(/|\+)[^/+]+)',
+    'enterprise-catalogs-course-detail',
```

Comment:
I have to make this change otherwise tests fails to find-out the end point because endpoint is expecting `(?P<course-key>[^/+]+(/|\\+)[^/+]+)` note escape backslash in regex.